tools/gce/linux_performance_worker_init.sh

```diff
@@ -88,6 +88,7 @@ curl -O https://bootstrap.pypa.io/get-pip.py
 sudo pypy get-pip.py
 sudo pypy -m pip install tabulate
 sudo pip install google-api-python-client
+sudo pip install tox
```

Comment:
this only adds it for pypy (installing pypy adds a new pip binary). Also add it a few lines above.